recipes/greedy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "greedyreg" %}
+{% set version = "1.0.1" %}
+{% set ref = "f10152c5374da08ee024c4c60ef8882876bd0808" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyushkevich/greedy/archive/{{ ref }}.tar.gz
+  sha256: 75386be7ca5495ce8a5115770cfe9ee838736d11008d792153bca57d81308919
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make
+  host:
+    - libitk-devel
+    - eigen
+    - vtk
+
+test:
+  commands:
+    - greedy -version
```

Comment:
It looks like you are shipping a C-API in addition to a CLI. If you are shipping a C-API, you need to do the following:

- [ ] Add existence tests for each of the classes of artifacts you are shipping: libraries, config files, headers
- [ ] Add run_exports to your package which will pin the appropriate ABI for downstream packages (please read the conda-build documentation if you are unfamiliar with run_exports).
- [ ] Remove static libraries or switch to shared libraries. Our channel policy prohibits shipping static libraries by default.